hoomd/md/pair/pair.py

```diff
@@ -428,9 +429,7 @@ class Yukawa(Pair):
         \end{eqnarray*}
 
     See `Pair` for details on how forces are calculated and the available
```

Comment:
Very minor nit, feel free to ignore this: the last word on this line, "available," is wrapped onto the next line in several of the Pair docstrings. Consistency might help future documentation writers that perform find/replace operations.